websites/V/vscode.dev/presence.ts

```diff
@@ -0,0 +1,626 @@
+const presence = new Presence({
+		clientId: "996589941926670366",
+	}),
+	startTimestamp = Math.floor(Date.now() / 1000);
+
+interface Stauts {
+	file: string;
+	branch: string;
+	error: string;
+	problems: string;
+	workspace: string;
+	editor: {
+		lang: string;
+		encoding: string;
+		selection: string;
+	};
+}
+
+/**
+ * Custom output
+ * - '%file%' (presence.ts)
+ * - '%branch%' (main)
+ * - '%error%' (0)
+ * - '%problems%' (0)
+ * - '%workspace%' (Presences)
+ * - '%lang%' (typescript)
+ * - '%encoding%' (UTF-8)
+ * - '%selection%' (Ln 20, Col 20)
+ */
```

Comment:
You can create a README.md file like Custom Status presence and add these customisations there